src/shared/firewall-util-nft.c

```diff
@@ -943,12 +943,16 @@ static int fw_nftables_add_local_dnat_internal(
                 const union in_addr_union *previous_remote) {
 
         sd_netlink_message *transaction[NFT_DNAT_MSGS] = {};
+        static bool ipv6_supported = true;
```

Comment:
maybe initialize this to `socket_ipv6_is_enabled()` actually, otherwise it's going to fail on such hosts sooner or later

Reply:
`_internal()` is not called when `socket_ipv6_is_enabled() == false`.